Sources/CodeEditTextView/STTextViewController.swift

```diff
@@ -221,6 +228,7 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
         rulerView?.backgroundColor = useThemeBackground ? theme.background : .clear
         rulerView?.separatorColor = theme.invisibles
         rulerView?.baselineOffset = baselineOffset
+        rulerView.isHidden = !lineNumbers
```

Comment:
worth noting that this change is within `reloadUI()` to provide the realtime updates